### ISSUE TITLE
Missing Convolution Subsampling?

### ISSUE THREAD

**Issue by fmac2000** [lucasnewman/best-rq-pytorch#1]
Hi Lucas,
I'm looking over the code and I believe you have missed the two convolution subsampling layers in conformer.py, 

> **4.1.1. NON-STREAMING MODELS** The model has two convolution layers at the bottom which provide 4 times temporal-dimension reduction for the input sequences. The rest of the layers are a stack of Conformer models. We explore 0.6B model size which is extensively studied in the previous works. The model contains 24 layers of Conformer models.

<img width="499" alt="Screenshot 2023-09-28 at 18 47 25" src="https://github.com/lucasnewman/best-rq-pytorch/assets/62116993/2103975e-f55d-449d-9f2d-6bfed394aa03">

If you'd like I can create a pull request and implement this for you now.
Thanks - If I've misunderstood the paper, please call me out! 😅

**Comment by lucasnewman** [comment]
@fmac2000 Yes, please, a PR would be great! I was aware of them in the paper but I skipped them for simplicity and downsampled in the feature extractor to get it off the ground. I would love to have it reflect the paper as closely as possible though!
